### PR TITLE
NO-JIRA: update library-go

### PR DESCRIPTION
**Pull request by tjungblu**
for openshift/library-go#1743

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,21 +6,21 @@ require (
...
-	k8s.io/apiserver v0.30.0
-	k8s.io/client-go v0.30.0
-	k8s.io/component-base v0.30.0
+	k8s.io/api v0.30.1
+	k8s.io/apimachinery v0.30.1
+	k8s.io/apiserver v0.30.1
+	k8s.io/client-go v0.30.1
+	k8s.io/component-base v0.30.1
 	k8s.io/klog/v2 v2.120.1
 	k8s.io/kube-scheduler v0.30.0
```
Comment (p0lyn0mial) [inline go.mod:25]: please update to `0.30.1`

Comment (tjungblu) [reply go.mod]: done

**Comment by p0lyn0mial** [comment]
/assign @ingvagabund 

for approval.

**Comment by ingvagabund** [comment]
Thank you
/approve

**Pull request merged**